src/util/click_helpers.py

```diff
@@ -0,0 +1,53 @@
+"""Helper to set the context settings for CLI commands that accept -h or
```

Comment:
Update this documentation to reflect that this file contains the click subclass that handles command aliasing, which is the main purpose of this file. We can still define `CONTEXT_SETTINGS` here, but that's not what the subclass does.